analyzers/tests/SonarAnalyzer.UnitTest/Rules/StreamReadStatementTest.cs

```diff
@@ -37,6 +37,10 @@ public class StreamReadStatementTest
         public void StreamReadStatement_CSharp10() =>
             builder.AddPaths("StreamReadStatement.CSharp10.cs").WithOptions(ParseOptionsHelper.FromCSharp10).Verify();
 
+        [TestMethod]
+        public void StreamReadStatement_CSharp11() =>
+            builder.AddPaths("StreamReadStatement.CSharp11.cs").WithOptions(ParseOptionsHelper.FromCSharp11).Verify();
```

Comment:
Technically the test could be added to the CSharp10 file, but because we are relying on .Net7 methods, which were released along with C#11, I added a new test file.